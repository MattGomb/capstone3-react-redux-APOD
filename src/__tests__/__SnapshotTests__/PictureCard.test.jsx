import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import store from '../../redux/ConfigureStore';
import Picture from '../../components/PictureCard';

describe('the PictureCard snapshot', () => {
  it('renders correctly', () => {
    const tree = render(
      <Provider store={store}>
        <Router>
          <Picture />
        </Router>
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});