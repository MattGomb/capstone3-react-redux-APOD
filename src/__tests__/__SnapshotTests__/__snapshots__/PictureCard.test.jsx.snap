// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the PictureCard snapshot renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="pictureCard"
      >
        <h2
          class="title"
        />
        <embed
          class="image"
        />
        <a
          href="/details/undefined"
        >
          <button
            class="detailBtn"
            type="submit"
          >
            Details
          </button>
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="pictureCard"
    >
      <h2
        class="title"
      />
      <embed
        class="image"
      />
      <a
        href="/details/undefined"
      >
        <button
          class="detailBtn"
          type="submit"
        >
          Details
        </button>
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
